docs/en/query_language/agg_functions/reference.md

```diff
@@ -1,12 +1,55 @@
-
 # Function reference
 
-## count() {#agg_function-count}
+## count {#agg_function-count}
+
+Counts the number of rows.
+
+```
+count()
+```
+
+ClickHouse supports the `COUNT(DISTINCT ...)` syntax sugar. The behavior of this construction depends on the [count_distinct_implementation](../../operations/settings/settings.md#settings-count_distinct_implementation) setting. It defines which of the `uniq*` functions is used to perform the operation.
```

Comment:
* `COUNT(DISTINCT ...)` is not a "sugar", this syntax is part of SQL standard and is way more widespread than separate aggregate functions, probably deserves separate explanation
* uniq functions could be a link
* it's worth to note that by default it's exact so people don't have to go through link to learn this